enslig-forsorger/src/main/kotlin/no/nav/familie/kontrakter/ef/iverksett/TekniskOpphørDto.kt

```diff
@@ -0,0 +1,19 @@
+package no.nav.familie.kontrakter.ef.iverksett
+
+import no.nav.familie.kontrakter.ef.felles.StønadType
+import java.time.LocalDate
+import java.util.UUID
+
+data class TekniskOpphørDto(
+        val forrigeBehandlingId: UUID,
+        val tilkjentYtelseMedMetaData: TilkjentYtelseMedMetaData
+)
+
+data class TilkjentYtelseMedMetaData(val saksbehandlerId: String,
+                                     val eksternBehandlingId: Long,
+                                     val stønadstype: StønadType,
+                                     val eksternFagsakId: Long,
+                                     val personIdent: String,
+                                     val behandlingId: UUID,
+                                     val vedtaksdato: LocalDate)
```

Comment:
Er det greit å bare inline denne i TekniskOpphørDto? 
Dette er veldig lik `TilkjentYtelseMedMetaData` som allerede finnes i iverksett men uten `tilkjentYtelse`, blir fort conflict med imports hvis vi har 2 av samme navn

Reply:
Vet ikke, tenkte det var enklere å bare legge de på toppnivå og uten å kalle det for noe med TilkjentYtelse

Reply:
Jo før all del, låt oss gå før TekniskOpphørDto